ui/components/ui/icon/icon.stories.js

```diff
@@ -163,6 +165,8 @@ export const DefaultStory = (args) => (
         <IconItem Component={<Preloader {...args} />} />
         <IconItem Component={<IconEye {...args} />} />
         <IconItem Component={<IconEyeSlash {...args} />} />
+        <IconItem Component={<CopyContractDetails {...args} />} />
+        <IconItem Component={<IconBlockExplorer {...args} />} />
```

Comment:
Seem like you are adding new icons to the DEPRECATED list move them up to current icons list

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/48c24613122a69ea20b0ad6504a3586585b6be25).